Sources/Impl/View+LTR.swift

```diff
@@ -23,7 +23,7 @@ import Foundation
 import UIKit
     
 extension UIView {
```

Comment:
Should probably be moved in the extension where protocol conformance is declared

Reply:
I agree, this should be moved in UIView+PinLayout.swift and NSView+PinLayout.swift